yarn.lock

```diff
@@ -10084,6 +10084,11 @@ jose@^1.28.1:
   dependencies:
     "@panva/asn1.js" "^1.0.0"
 
+jose@^4.10.0:
```

Comment:
jose is added,to global lock file, but it isn't a dep anymore

Reply:
removed